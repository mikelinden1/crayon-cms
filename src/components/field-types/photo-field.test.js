import PhotoField from './photo-field';

const props = {
    name: 'test',
    value: 'val1',
    label: 'Test',
    onChange: () => {},
    showMediaPicker: () => {},
    disabled: false
};

it('PhotoField matches snapshot', () => {
    const wrapper = shallow(<PhotoField {...props} />);
    expect(wrapper).toMatchSnapshot();
});
