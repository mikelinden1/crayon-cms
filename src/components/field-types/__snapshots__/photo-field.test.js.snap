// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoField matches snapshot 1`] = `
<div
  className="photo-field input-group"
>
  <input
    className="form-control"
    disabled={false}
    name="test"
    onChange={[Function]}
    type="text"
    value="val1"
  />
  <span
    className="input-group-btn"
  >
    <Button
      className="btn-sm"
      color="primary"
      disabled={false}
      onClick={[Function]}
      tag="button"
    >
      Select
    </Button>
  </span>
</div>
`;
